### PR TITLE
WP-85 Design Videos player page

### PR DESCRIPTION
**Pull request by ericgmoon**
Using the `VideoPage` component for the purpose of the PR, until the nav page for videos is implemented.

Screenshots:
image
image
image

### CODE REVIEW

File: src/pages/VideosPage/index.tsx [hunk 1]
```diff
@@ -1,10 +1,232 @@
...
+  ListIndexLabel,
+  ListLabel,
+  PlayerColumn,
+  SidebarHeader,
+  SidebarHeaderWrapper,
+  TitleBarContainer,
+  UtilColumn,
+} from './index.styles';
+
+const formatDate = (date: Date) => {
```
Comment (s-hane) [inline src/pages/VideosPage/index.tsx:32]: Should this not be in utils, isn't this used for tutorials too?

File: src/pages/VideosPage/index.tsx [hunk 2]
```diff
@@ -1,10 +1,232 @@
...
+  href: string,
+}
+
+interface SideBarProps {
+  resources?: Resource[],
+  relatedVideos?: Video[],
+  currentVideoId: string,
+}
+
+const SideBar = ({ resources = [], relatedVideos = [], currentVideoId } : SideBarProps) => (
```
Comment (s-hane) [inline src/pages/VideosPage/index.tsx:74]: Should this not be a separate component? Can't this be used for notes as well?

File: src/pages/VideosPage/index.tsx (hunk 1 above)
Comment (ericgmoon) [reply src/pages/VideosPage/index.tsx:32]: Nah, the date format is different

Comment (ericgmoon) [reply src/pages/VideosPage/index.tsx:32]: Also - that would make it hard to change the date format in one area exclusively

File: src/pages/VideosPage/index.tsx (hunk 2 above)
Comment (ericgmoon) [reply src/pages/VideosPage/index.tsx:74]: I think it'd be quite different but let's see when we make the Notes page

**Pull request merged**